controllers/clusterversion_controller.go

```diff
@@ -320,6 +348,22 @@ func (c *ClusterVersionReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 	return ctrl.Result{}, nil
 }
 
+func (c *ClusterVersionReconciler) deletionPhase() (reconcile.Result, error) {
+	if err := csi.DeleteCSIDriver(c.ctx, c.Client, csi.GetCephFSDriverName()); err != nil && !k8serrors.IsNotFound(err) {
+		c.log.Error(err, "unable to delete cephfs CSIDriver")
+		return ctrl.Result{}, err
+	}
+	if err := csi.DeleteCSIDriver(c.ctx, c.Client, csi.GetRBDDriverName()); err != nil && !k8serrors.IsNotFound(err) {
```

Comment:
where are these `DeleteCSIDriver` methods defined?

Reply:
The `DeleteCSIDriver` is a function that was already in the repo under `pkg/csi/`